code/obj/item/sec_tape.dm

```diff
@@ -0,0 +1,181 @@
+			T1.layerify()
+			var/obj/sec_tape/T2 = new /obj/sec_tape(T, src)
+			T2.dir = WEST
+			T2.tape_UpdateIcon()
+			T2.layerify()
+			var/obj/sec_tape/T3 = new /obj/sec_tape(T, src)
+			T3.dir = NORTH
+			T3.tape_UpdateIcon()
+			T3.layerify()
+			var/obj/sec_tape/T4 = new /obj/sec_tape(T, src)
+			T4.tape_UpdateIcon()
+			T4.layerify()
+			//No need to set it up to the south, it already is by default
+			user.death(FALSE)
+			qdel(src)
+		else
+			//Else we'll choke ourselves out
+			user.visible_message("<span class='alert'><b>[user] wraps some tape around \his neck and tightens it.</b></span>")
+			user.take_oxygen_deprivation(160)
+			SPAWN(50 SECONDS)
+				if (user && !isdead(user))
+					user.suiciding = 0
+		return 1
+
+	proc/use(var/used) // remove "used" amount from the coil
+		if (src.amount < used)
+			return 0
+		else if (src.amount == used)
+			qdel(src)
+			return 1
+		else
+			amount -= used
+			tooltip_rebuild = 1
+			src.UpdateIcon()
+			return 1
+
+	proc/take(var/amt, var/newloc) // removes "amt" from the coil and put it somewhere, use for coil splitting with wirecutters
+		if (amt > amount)
+			amt = amount
+			tooltip_rebuild = 1
+			src.UpdateIcon()
+		if (amt == amount)
+			if (ismob(loc))
+				var/mob/owner = loc
+				owner.u_equip(src)
+			set_loc(newloc)
+			return src
+		src.use(amt)
+		var/obj/item/sec_tape/C = new /obj/item/sec_tape(newloc)
+		C.amount = amt
+		return
+
+	update_icon()
+		inventory_counter?.update_number(amount)
+		return
+
+/obj/item/sec_tape/vended //Tape in secvends has a pre-set amount of uses
+	New(loc, length)
+		..(loc, 20)
+
+/obj/item/sec_tape/get_desc()
+	return "There's [amount] length[s_es(amount)] left."
+
+/obj/item/sec_tape/attackby(obj/item/W, mob/user)
+	if (issnippingtool(W) && src.amount > 1) //Cut some of it off to share
+		take(1, user.loc)
+		tooltip_rebuild = 1
+		boutput(user, "You cut a piece off the [base_name].")
+		src.UpdateIcon()
+		return
+
+	else if (istype(W, /obj/item/sec_tape))
+		var/obj/item/sec_tape/C = W
+
+		if (C.amount == MAXTAPE)
+			boutput(user, "The coil is too long, you cannot add any more tape to it.")
+			return
+
+		if ((C.amount + src.amount <= MAXTAPE))
+			C.amount += src.amount
+			boutput(user, "You join the tape ends together.")
+			C.tooltip_rebuild = 1
+			C.UpdateIcon()
+			if(istype(src.loc, /obj/item/storage))
+				var/obj/item/storage/storage = src.loc
+				storage.hud.remove_object(src)
+			else if(istype(src.loc, /mob))
+				var/mob/M = src.loc
+				M.u_equip(src)
+				M.drop_item(src)
+			qdel(src)
+			return
+
+		else
+			boutput(user, "You transfer [MAXTAPE - src.amount] length\s of tape from one roll to the other.")
+			src.amount -= (MAXTAPE-C.amount)
+			src.UpdateIcon()
+			tooltip_rebuild = 1
+			C.amount = MAXTAPE
+			C.UpdateIcon()
+			C.tooltip_rebuild = 1
+			return
+
+/obj/item/sec_tape/MouseDrop_T(atom/movable/O as obj, mob/user as mob)
+	..(O, user)
+	for (var/obj/item/sec_tape/C in view(1, user))
+		C.UpdateIcon()
```

Comment:
What's this supposed to do?

Reply:
Removed it, seems it didnt actually do anything.